### ISSUE TITLE
Access to the inference results for reproduction of learning unified label space

### ISSUE THREAD

**Issue by yuleichin** [xingyizhou/UniDet#20]
Hi, xingyi,
Thanks for your work.
I would like to reproduce the label space results, but it seems my inference json files are not appropriate for learning the unified space. Would you please help provide some examples of your inference results (json format) so that I can try if my codes went wrong. For example, the data path in </tools/UniDet_learn_labelspace_mAP.ipynb>:
```
DATA_PATH = {
    'objects365': ROOT_PATH + 'logits/{}/inference_objects365_val/unified_instances_results.json'.format(EXP_NAME),
    'coco': ROOT_PATH + 'logits/{}/inference_coco_2017_val/unified_instances_results.json'.format(EXP_NAME),
    'oid': ROOT_PATH + 'logits/{}/inference_oid_val_expanded/unified_instances_results.json'.format(EXP_NAME),
}
```
I appreciate it a lot if you can share the three json files above. Thank you very much!
